gis/gccview/gcc_puller_functions.py

```diff
@@ -451,6 +498,58 @@ def insert_partitioned_data(schema_parent_table_insert, insert_column, return_js
         with con.cursor() as cur:
                execute_values(cur, insert, rows)
     LOGGER.info('Successfully inserted %d records into %s', len(rows), schema_parent_table_insert)
+    
+def insert_data(output_table, insert_column, return_json, schema_name, con):
```

Comment:
I'm going to be picky and ask you to combine the 3 insert_data functions. They each have just 1-2 lines different.